lib/Scheduler/list_sched.cpp

```diff
@@ -47,6 +63,26 @@ FUNC_RESULT ListScheduler::FindSchedule(InstSchedule *sched, SchedRegion *rgn) {
       maxRdyLstSize = rdyLstSize;
     avgRdyLstSize += rdyLstSize;
 
+
+
+    SchedInstruction *inst;
+
+    // TODO -- extract this into config variable
+    bool forcedSchedule = false;
+
+    // Force get the schedule in order of best heuristic value (not just best
+    // available/ready)
+
+    if (forcedSchedule) {
+      if (numToPick == -1 || CheckForInst(numToPick)) {
+        inst = PickInst();
+        assert(inst);
+        if (numToPick == -1) entry = inst->GetNum();
+        numToPick += 1;
+        if (numToPick == entry) numToPick += 1;
+      }
+    }
+
     SchedInstruction *inst = PickInst();
```

Comment:
```suggestion
    inst = PickInst();
```
inst is already defined above on line 68.

Reply:
Thanks for picking up on those things -- I ran into the same issues, but I suppose I didn't update the PR